spyder/utils/introspection/utils.py

```diff
@@ -210,6 +210,15 @@ def get_keywords(lexer):
                 continue
     return keywords
 
+def get_words(text_from_file):
+    """ Completion inline
+    Extract the list of words that contains in text_from_file,
+    to carry out the inline completion similar to VSCode.
+    """
+    lines = [re.sub(r'([^a-zA-Z-_])', ' ', line, flags=re.UNICODE).split() \
+             for line in text_from_file.split("\n")]
```

Comment:
As a general rule If you cannot fit a list comprehension in a single line, then you should not use one.

Just use a normal for that is more readable.

Also using `   \   ` to join code lines is discouraged as a style guide.